src/app/component/user/components/profile/profile-widget/profile-header/profile-header.component.html

```diff
@@ -23,7 +23,9 @@
     <div class="rate">
       <p>{{ 'profile.rate' | translate }}: {{ userInfo.rating || mockedUserInfo.rating }}</p>
     </div>
-    <div class="credo">{{ userInfo.userCredo || mockedUserInfo.userCredo }}</div>
+    <div *ngIf="userInfo.userCredo?.length"
```

Comment:
what if userInfo will be undefined ? what than "*ngIf="undefined"" will produce ? error ?

Reply:
So, you mean to change this line of code to 'userInfo.userCredo?.length && userInfo != undefined' ?

Reply:
i think we need to move it getter and do a check there... 
get something () { 
if (userInfo && userInfo.userCredo) {return userInfo.userCredo.length}  
} return 0

Reply:
Roger that